fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1327,6 +1332,7 @@ public void loadImage(String imageDir) throws IOException, DdlException {
             remoteChecksum = dis.readLong();
             checksum = MVManager.getInstance().reload(dis, checksum);
             remoteChecksum = dis.readLong();
+            nodeMgr.loadGlobalFunctions(dis, checksum);
```

Comment:
Why use NodeMgr to loadGlobalFunctions and saveGlobalFunctions? I thinks it's better to put these at GlobalFunctionMgr

Reply:
Yes, it's much better.